crates/consensus/Cargo.toml

```diff
@@ -20,14 +20,15 @@ alloy-rlp.workspace = true
 alloy-eips.workspace = true
 alloy-consensus.workspace = true
 alloy-primitives = { workspace = true, features = ["rlp"] }
+alloy-json-rpc.workspace = true
```

Comment:
I think this one could be problematic because:

https://github.com/alloy-rs/alloy/blob/e94b8d4439fd5e28da354d185e68623ec0b1db2e/crates/json-rpc/Cargo.toml#L28-L29

Reply:
yeah we can do that :)